x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityEsqlIT.java

```diff
@@ -73,10 +76,13 @@ public class RemoteClusterSecurityEsqlIT extends AbstractRemoteClusterSecurityTe
 
         queryCluster = ElasticsearchCluster.local()
             .name("query-cluster")
+            .module("x-pack-autoscaling")
             .module("x-pack-esql")
             .module("x-pack-enrich")
+            .module("x-pack-ml")
             .module("ingest-common")
             .apply(commonClusterConfig)
+            .setting("xpack.ml.enabled", "false")
```

Comment:
I remember I did that because of some locally running tests issues. You did this because of this or some other reason?

Reply:
To implement this ES|QL categorize text feature, the `esql` plugin has to depend on the `ml` plugin, which contains the categorize text code. If `ml` is fully enabled, it needs a writeable tmp dir, a bunch of named pipes etc. Part of that fails in this IT setup.